activities.py

```diff
@@ -289,7 +289,7 @@ def clean_tag(tag):
         """
         Return a BeautifulSoup's tag contents as a string.
         """
-        return "".join(tag.stripped_strings).replace("\n", " ")
+        return re.sub("\n\s*", " ", tag.get_text()).strip()
```

Comment:
Is it better to use `[\n\s]*`?  Or even `[\r\n\s]*`.  Or can we rely on the tags having whitespace collapsed and CRLF-normalized to `\n`?

Reply:
So I was intending to keep double-space after a period or colon when it was present -- although I wouldn't then create one at end-of-line, and this managed to do that, whereas replacing all whitespace with a single space wouldn't.  And as far as I can tell, (a) the only place there's odd whitespace we want to get rid of is at start-of-line, and (b) I haven't run into any `\r`s.  But I may run into odder specs in the future...

And note that `\n` is included within `\s`, which means `[ \t\n\r\f\v]` (or more when `UNICODE` is specified).  See https://docs.python.org/2/library/re.html .